polaris/ocean/convergence/forward.py

```diff
@@ -112,20 +129,35 @@ def dynamic_model_config(self, at_setup):
             self.add_yaml_file('polaris.ocean.config', 'single_layer.yaml')
 
         section = config['convergence_forward']
-
         time_integrator = section.get('time_integrator')
-
-        # dt is proportional to resolution: default 30 seconds per km
-        if time_integrator == 'RK4':
-            dt_per_km = section.getfloat('rk4_dt_per_km')
+        if self.dt is None:
+            # dt is proportional to resolution: default 30 seconds per km
+            if time_integrator == 'RK4':
+                self.dt = section.getfloat('rk4_dt_per_km')
+            elif time_integrator == 'FB_LTS':
+                self.dt = section.getfloat('fb_lts_dt_per_km')
+            elif time_integrator == 'LTS':
+                self.dt = section.getfloat('lts_dt_per_km')
+            elif time_integrator == 'unsplit_explicit':
+                self.dt = section.getfloat('unsplit_explicit_dt_per_km')
+            elif time_integrator == 'split_implicit':
+                self.dt = section.getfloat('split_implicit_dt_per_km')
+            elif time_integrator == 'split_explicit_ab2':
+                self.dt = section.getfloat('split_explicit_ab2_dt_per_km')
+            else:
+                raise ValueError('Time integrator selected is not available.')
+
+            dt_str = get_time_interval_string(seconds=self.dt *
+                                              self.resolution)
+            dt_btr_scaling = section.getfloat('dt_btr_scaling')
+            dt_btr = self.dt / dt_btr_scaling
+            btr_dt_str = get_time_interval_string(seconds=dt_btr *
+                                                  self.resolution)
```

Comment:
I don't think we can make this change without also updating any test cases that already use the split time stepping to use this approach.  Maybe none are so far in which case maybe this will be fine.